README.md

```diff
@@ -49,47 +49,64 @@ $client = new Client(new Settings(AuthToken::generate('username', 'password', 'd
 And then we can make next operations _(each call will returns an object with server response data)_:
 
 ```php
-<?php /** @var \Avtocod\B2BApi\Client $client */
+<?php
+
+use \Avtocod\B2BApi\Client;
+use \Avtocod\B2BApi\Params\DevPingParams;
+use \Avtocod\B2BApi\Params\DevTokenParams;
+use \Avtocod\B2BApi\Params\UserParams;
+use \Avtocod\B2BApi\Params\UserBalanceParams;
+use \Avtocod\B2BApi\Params\UserReportTypesParams;
+use \Avtocod\B2BApi\Params\UserReportsParams;
+use \Avtocod\B2BApi\Params\UserReportParams;
+use \Avtocod\B2BApi\Params\UserReportMakeParams;
+use \Avtocod\B2BApi\Params\UserReportRefreshParams;
+
+/** @var Client $client */
 
 // Test connection
-$client->devPing();
+$client->devPing(new DevPingParams());
```

Comment:
```suggestion
$client->devPing(new DevPingParams);
```

You can ommit `()` here and below

Reply:
In all places, removed `()` if constructor is empty adbb3ff001563e430a08eca13f4bf7da54122c99

Reply:
Removed in all places in commit adbb3ff001563e430a08eca13f4bf7da54122c99